src/com_tjvendors/admin/tjvendorsTemplate.json

```diff
@@ -180,27 +236,41 @@
     "client": "com_tjvendors",
     "key": "approvalOnVendorMailToOwner",
     "title": "Admin approves Vendor (approval On)",
-    "email_body": "<p>Dear {vendor_user.name},<br /><br />Your vendor profile <strong>{vendor_data.vendor_title}</strong> has been approved by admin. Now you can further continue.<br /><br />This is system generated email, please do not reply. Many thanks from {info.sitename}</p>",
-    "sms_body": "",
-    "push_body": "",
-    "web_body": "",
-    "email_subject": "Your vendor registration for \"{vendor_data.vendor_title}\" at {info.sitename} has been approved!",
-    "sms_subject": "",
-    "push_subject": "",
-    "web_subject": "",
-    "email_status": 1,
-    "sms_status": 0,
-    "push_status": 0,
-    "web_status": 0,
     "user_control": 1,
+    "state": 1,
     "core": 1,
+    "email":{
+		"state": 1,
+		"cc": "",
+		"bcc": "",
+		"from_name": "",
+		"from_email": "",
+		"emailfields": {
+			"emailfields0": {
+				"id": "",
+				"language": "*",
+				"subject": "Your vendor registration for \"{{vendor_data.vendor_title}}\" at {{info.sitename}} has been approved!",
+				"body": "<p>Dear {{vendor_user.name}},<br /><br />Your vendor profile <strong>{{vendor_data.vendor_title}}</strong> has been approved by admin. Now you can further continue.<br /><br />This is system generated email, please do not reply. Many thanks from {{info.sitename}}</p>"
+			}
+		}
+	},
+	"sms": {
+		"state": 1,
+		"smsfields": {
+			"smsfields0": {
+				"id": "",
+				"language": "*",
+				"body": "Dear {{vendor_user.name}}, Your vendor profile {{vendor_data.vendor_title}} has been approved by admin. Now you can further continue."
```

Comment:
change as


Dear {{vendor_user.name}}, Your vendor profile {{vendor_data.vendor_title}} has been approved by admin. Now you can continue further.